arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,778 @@
+// SPDX-License-Identifier: GPL-2.0+ or X11
+/*
+ * Copyright (C) 2021-2023 Technologic Systems, Inc. dba embeddedTS
+ */
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include "imx6ul.dtsi"
+
+/ {
+	model = "embeddedTS i.MX6ul TS-7180";
+	compatible = "technologic,ts7180", "fsl,imx6ul";
+
+	aliases {
+		ethernet0 = &fec2;
+		ethernet1 = &fec1;
+	};
```

Comment:
Does this actually do the correct ordering? I feel like I tried this on other platforms and it still kept the reversed numbers.